### PR TITLE
[DET-706] Add an AUTO permission for webhook auth and CLI stuff for generating webhook

### PR DESCRIPTION
**Pull request by khyberspache**

### CODE REVIEW

File: javascript/sdk/lib/controllers/partner.ts [hunk 1]
```diff
@@ -112,4 +112,20 @@ export default class PartnerController {
...
+  ): Promise<StatusResponse> {
+    const response = await this.#client.requestWithAuth(
+      `/partner/suppress/${partnerCode}`,
+      {
+        method: "GET",
+        ...options,
+      }
+    );
+
+    return response.json();
```
Comment (stephanwampouille) [inline javascript/sdk/lib/controllers/partner.ts:129]: this might need to be awaited?

```suggestion
    return await response.json();
```

**Review by stephanwampouille — CHANGES_REQUESTED** [review]

File: javascript/sdk/lib/controllers/partner.ts (hunk 1 above)
Comment (khyberspache) [reply javascript/sdk/lib/controllers/partner.ts:129]: Make edits as necessary

Comment (stephanwampouille) [reply javascript/sdk/lib/controllers/partner.ts]: updated btw

File: python/cli/prelude_cli/views/partner.py [hunk 2]
```diff
@@ -81,3 +81,14 @@ def partner_deploy(controller, partner, host_ids):
     with Spinner(description='Deploying probes to hosts'):
         data = controller.deploy(partner_code=Control[partner.upper()].value, host_ids=host_ids)
     print_json(data=data)
+
+
+@partner.command('webhook-auth')
+@click.argument('partner', type=click.Choice([c.name for c in Control if c != Control.INVALID], case_sensitive=False))
```
Comment (privateducky) [inline python/cli/prelude_cli/views/partner.py:87]: should this be restricted to EDR controls instead? or do you think doesn't matter/too niche?

File: python/cli/prelude_cli/views/partner.py [hunk 3]
```diff
@@ -81,3 +81,14 @@ def partner_deploy(controller, partner, host_ids):
...
+@click.pass_obj
+@handle_api_error
+def feed_auth(controller, partner):
+    """ Get feed authentication information for a partner """
```
Comment (privateducky) [inline python/cli/prelude_cli/views/partner.py:91]: can you add a better comment, more user-friendly?

Comment (privateducky) [reply python/cli/prelude_cli/views/partner.py:91]: same with the command; i'd use something like ``webook-create`` or ``webhook-generate``. it's more descriptive of what you're doing in totality.. then i'd probably put a print line that points to our docs on how to finish the work/next steps.

**Review by privateducky — CHANGES_REQUESTED** [review]
couple small items

File: python/cli/prelude_cli/views/partner.py (hunk 2 above)
Comment (khyberspache) [reply python/cli/prelude_cli/views/partner.py:87]: Yeah that's good point - I think it's worth adding the check. I'll do that

File: python/cli/prelude_cli/views/partner.py (hunk 3 above)
Comment (khyberspache) [reply python/cli/prelude_cli/views/partner.py:91]: I think webhook-generate is a good option. I'll shift to that

**Review by privateducky — APPROVED** [review]

**Pull request merged**